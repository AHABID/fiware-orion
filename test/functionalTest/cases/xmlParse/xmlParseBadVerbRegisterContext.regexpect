HTTP/1.1 405 Method Not Allowed
Content-Length: 0
Allow: POST
Date: REGEX(.*)

++++++++++++++++++++
