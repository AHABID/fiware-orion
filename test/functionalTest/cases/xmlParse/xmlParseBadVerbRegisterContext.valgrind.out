==28800== Memcheck, a memory error detector
==28800== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==28800== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==28800== Command: contextBroker -vvvvv -harakiri -port 9999 -pidpath /tmp/orion_9999.pid -db testharness -t 0-255
==28800== 
T@09:29:03  paConfigActions[727]: setting trace levels to '0-255'
T@09:29:03  paConfigActions[728]: Leaving function paConfigActions
E@09:29:03  exitFunc[559]: unlink(/tmp/orion_9999.pid): No such file or directory
T@09:29:03  semInit[61]: Initialized 'req' semaphore
T@09:29:03  semInit[62]: Initialized 'mongo' semaphore
T@09:29:03  mongoSemTake[90]: mongoConnect taking the 'mongo' semaphore for 'connecting to mongo'
T@09:29:03  mongoSemTake[92]: mongoConnect has the 'mongo' semaphore
==28800== 
==28800== HEAP SUMMARY:
==28800==     in use at exit: 17,690 bytes in 104 blocks
==28800==   total heap usage: 3,269 allocs, 3,165 frees, 313,863 bytes allocated
==28800== 
==28800== LEAK SUMMARY:
==28800==    definitely lost: 0 bytes in 0 blocks
==28800==    indirectly lost: 0 bytes in 0 blocks
==28800==      possibly lost: 0 bytes in 0 blocks
==28800==    still reachable: 17,690 bytes in 104 blocks
==28800==         suppressed: 0 bytes in 0 blocks
==28800== Rerun with --leak-check=full to see details of leaked memory
==28800== 
==28800== For counts of detected and suppressed errors, rerun with: -v
==28800== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 7 from 7)
T@09:29:03  mongoConnect[136]: mongo version server: 2.4.9 (mayor: 2, minor: 4, extra: 9)
T@09:29:03  mongoSemGive[118]: mongoConnect gives the 'mongo' semaphore for 'connecting to mongo'
F@09:29:03  ***[600]: Connected to mongo at localhost:testharness
T@09:29:03  ensureLocationIndex[336]: ensuring 2dsphere index on location.coords
T@09:29:03  recoverOntimeIntervalThreads[353]: query() in 'testharness.csubs' collection: '{ conditions.type: "ONTIMEINTERVAL" }'
T@09:29:03  mongoSemTake[90]: recoverOntimeIntervalThreads taking the 'mongo' semaphore for 'query in SubscribeContextCollection'
T@09:29:03  mongoSemTake[92]: recoverOntimeIntervalThreads has the 'mongo' semaphore
T@09:29:03  mongoSemGive[118]: recoverOntimeIntervalThreads gives the 'mongo' semaphore for 'query in SubscribeContextCollection'
==28805== Source and destination overlap in strncpy(0xa4d8a7, 0xa4d8a7, 63)
==28805==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==28805==    by 0x659678: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:600)
==28805==    by 0x5DF38D: main (contextBroker.cpp:737)
==28805== 
==28805== Source and destination overlap in strncpy(0xa4d8e2, 0xa4d8e2, 63)
==28805==    at 0x4C29782: strncpy (mc_replace_strmem.c:463)
==28805==    by 0x65969C: restInit(RestService*, IpVersion, char const*, unsigned short, char const*, char const*, void (*)(ConnectionInfo*), bool) (rest.cpp:603)
==28805==    by 0x5DF38D: main (contextBroker.cpp:737)
==28805== 
V@09:29:03  restStart[499]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@09:29:03  restStart[541]: Starting HTTP daemon on IPv6 :: port 9999
==28805== Thread 4:
==28805== Syscall param mmap(length) contains uninitialised byte(s)
==28805==    at 0x58B9AFA: mmap (syscall-template.S:82)
==28805==    by 0x4E398E7: MHD_pool_create (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35F47: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35F20: MHD_connection_get_pollfd (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35FFB: MHD_connection_get_fdset (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E373B0: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E33C62: MHD_set_connection_value (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34033: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E3426D: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34783: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
T@09:29:11  httpHeaderGet[94]: HTTP Header:   User-Agent: curl/7.26.0
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Host: localhost:9999
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Content-Type: application/xml
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Accept: application/xml
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Content-Length: 1155
T@09:29:11  restService[153]: Treating service * /ngsi10/subscribeContext
W@09:29:11  badVerbPostOnly[48]: bad verb for url '/ngsi10/subscribeContext', method 'BAD_VERB'
T@09:29:11  restReply[65]: Response 1: responding with 0 bytes, Status Code 405
T@09:29:11  restReply[66]: Response payload: ''
T@09:29:11  ~RegisterContextResponse[56]: destroyed
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E399BC: MHD_pool_allocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E335B6: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E335BD: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2A639: memcpy (mc_replace_strmem.c:121)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2A64E: memcpy (mc_replace_strmem.c:838)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2A69B: memcpy (mc_replace_strmem.c:838)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2A790: memcpy (mc_replace_strmem.c:838)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Use of uninitialised value of size 8
==28805==    at 0x4C2A7C8: memcpy (mc_replace_strmem.c:838)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Use of uninitialised value of size 8
==28805==    at 0x4C2A7ED: memcpy (mc_replace_strmem.c:838)
==28805==    by 0x4E338FA: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Use of uninitialised value of size 8
==28805==    at 0x58CFC9C: __vsprintf_chk (vsprintf_chk.c:79)
==28805==    by 0x58CFC0E: __sprintf_chk (sprintf_chk.c:33)
==28805==    by 0x4E33622: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Use of uninitialised value of size 8
==28805==    at 0x4C2953C: strcpy (mc_replace_strmem.c:429)
==28805==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C29569: strcpy (mc_replace_strmem.c:112)
==28805==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2956E: strcpy (mc_replace_strmem.c:121)
==28805==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4C2957F: strcpy (mc_replace_strmem.c:429)
==28805==    by 0x4E33913: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Use of uninitialised value of size 8
==28805==    at 0x4E3363E: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E34DE0: MHD_connection_handle_idle (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37420: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Syscall param sendto(msg) contains uninitialised byte(s)
==28805==    at 0x58BE0AC: send (send.c:32)
==28805==    by 0x4E36D69: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35963: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35A87: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A40: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E33BA1: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35A9D: MHD_connection_handle_write (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37638: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A1F: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A24: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Conditional jump or move depends on uninitialised value(s)
==28805==    at 0x4E39A52: MHD_pool_reallocate (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E334BF: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E35D17: MHD_connection_handle_read (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37648: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
==28805== Thread 3:
==28805== Syscall param munmap(length) contains uninitialised byte(s)
==28805==    at 0x58B9B27: munmap (syscall-template.S:82)
==28805==    by 0x4E3996B: MHD_pool_destroy (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E37030: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x4E3881C: ??? (in /usr/lib/x86_64-linux-gnu/libmicrohttpd.so.10.15.0)
==28805==    by 0x5B72B4F: start_thread (pthread_create.c:304)
==28805==    by 0x58BD0EC: clone (clone.S:112)
==28805== 
T@09:29:11  httpHeaderGet[94]: HTTP Header:   User-Agent: curl/7.26.0
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Host: localhost:9999
T@09:29:11  httpHeaderGet[94]: HTTP Header:   Accept: */*
T@09:29:11  restService[153]: Treating service GET /exit/harakiri
E@09:29:11  orionExit[548]: Received a 'DIE' request
==28805== 
==28805== HEAP SUMMARY:
==28805==     in use at exit: 23,191 bytes in 148 blocks
==28805==   total heap usage: 3,528 allocs, 3,380 frees, 1,256,122 bytes allocated
==28805== 
==28805== LEAK SUMMARY:
==28805==    definitely lost: 0 bytes in 0 blocks
==28805==    indirectly lost: 0 bytes in 0 blocks
==28805==      possibly lost: 1,394 bytes in 16 blocks
==28805==    still reachable: 21,797 bytes in 132 blocks
==28805==         suppressed: 0 bytes in 0 blocks
==28805== Rerun with --leak-check=full to see details of leaked memory
==28805== 
==28805== For counts of detected and suppressed errors, rerun with: -v
==28805== Use --track-origins=yes to see where uninitialised values come from
==28805== ERROR SUMMARY: 44 errors from 29 contexts (suppressed: 7 from 7)
